doc/supported_formats/delmic.rst

```diff
@@ -15,8 +19,22 @@ RosettaScIO can only read hyperspectral cathodoluminescence .h5 datasets from De
         >>> import hyperspy.api as hs
         >>> hs.load("filename.h5", reader="Delmic")
 
+By default, all datasets found in the acquisition files are opened with RosettaSciIO.
+As typically Delmic CL data acquisitions contain three datasets, you will obtain a list
+of three datasets. It is possible to load each of them separately.
+
+.. Note::
+    To load the various types of datasets in the file, use the ``signal`` argument
```

Comment:
We were discussing what is actually the best default here - reading only CL to return a single signal item as in most other readers or the list with all items, but I guess the main point would be that it is well documented.

Also, I am not certain having the survey first is the best order of the signals in the list. I assume the order is motivated by the odemis files? Generally, I would find it more intuitive to have the actual CL signal first, then concurrent and last survey (making the survey always last item, even if there are multiple streams in one file).

Finally, I would say there is not really a point in reading in the concurrent `se` when it is dimensionless (for spectra, single transients or streak images).

Reply:
Fair enough for the datasets with a single point. Only, I would still tend to reverse the order of the different signals in the list for "all" to have the 'cl' first (as it is the main one and now also the one that is read by default) and then add the additional signals with 'survey' last.